Tasks/Chef/Chef.ps1

```diff
@@ -153,7 +153,7 @@ try
     $connectedServiceDetails = Get-ConnectedServiceDetails -Context $distributedTaskContext -ConnectedServiceName $connectedServiceName
 
 	#setting up chef repo with the chef subscription details fetched before
-    Setup-ChefRepo $connectedServiceDetails
+    Initialize-ChefRepo $connectedServiceDetails
```

Comment:
This breaks compat with old Agent. Should you call out Agent version required for this Task.